test/e2e/network_create_test.go

```diff
@@ -56,9 +56,16 @@ func genericPluginsToPortMap(plugins interface{}, pluginType string) (network.Po
 }
 
 func (p *PodmanTestIntegration) removeCNINetwork(name string) {
-	session := p.Podman([]string{"network", "rm", "-f", name})
+	// Some tests remove the network manual but they should
+	// still defer this function in order to prevent flakes.
+	// So only remove the network if it exists.
+	session := p.Podman([]string{"network", "inspect", name})
 	session.WaitWithDefaultTimeout()
-	Expect(session.ExitCode()).To(BeZero())
+	if session.ExitCode() == 0 {
+		session = p.Podman([]string{"network", "rm", "-f", name})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(BeZero())
+	}
```

Comment:
Sorry for the late review; I had a concern and wanted to sleep on it.

My concern is that many of the e2e tests are already hard to read, and `podman network inspect` will add thirty more lines to already-long output. What would you think of just always force-running `podman network rm -f`, and just not bothering to check exit status? This is cleanup, not a test: the tests that need to check `network rm` already do so explicitly. 

That leaves the possibility that `network rm -f` could fail for a reason other than `network not found`. I don't know how likely that is, or how serious. (I suspect that it would lead to further failures that would be caught, but don't really know). So, I'm not filing this as a please-change, more like a please-discuss. @mheon @giuseppe @rhatdan WDYT?

Reply:
Well for consistency we could make podman rm network exit with 1 when the name does not exists like we do with containers.
```
$ podman network rm foobar1; echo $?
Error: unable to find network configuration for foobar1: network not found
125
$ podman container rm foobar1; echo $?
Error: no container with name or ID foobar1 found: no such container
1
podman image rm foobar1; echo $?
Error: 1 error occurred:
	* unable to find a name and tag match for foobar1 in repotags: no such image
1

```
Then we could check if the exit code was greater then 1?

Reply:
Oh, I really like that. 125 troubles me, because it (should be) reserved for podman-internal errors.